tests/integration_tests/performance/test_network_latency.py

```diff
@@ -180,10 +181,26 @@ def _g2h_send_ping(context):
                               ssh_key=ssh_key,
                               config=context.microvm,
                               network_config=network_config)
-
     _tap, host_ip, _ = basevm.ssh_network_config(network_config, '1')
 
     basevm.start()
+
+    # Pin uVM threads to physical cores.
+    match = re.match("([0-9]+)vcpu_[0-9]+mb", context.microvm.name())
```

Comment:
Can't we find the number of vcpus in a different way ? Do we have access to the microvm macine config for example ?

Reply:
We can issue a `nproc` on the guest. I'll try it and let you know how it goes.

Reply:
Can we implement an accessor within the microvm() class so that we pick up the number of cpus set at startup?

Reply:
This sounds like a really nice approach. I'll try this.